src/bokeh/util/paths.py

```diff
@@ -18,52 +18,53 @@
 #-----------------------------------------------------------------------------
 
 # Standard library imports
-import sys
-from os.path import (
-    abspath,
-    dirname,
-    isdir,
-    join,
-    normpath,
-    realpath,
-)
+from pathlib import Path
 
 #-----------------------------------------------------------------------------
 # Globals and constants
 #-----------------------------------------------------------------------------
 
 # Root dir of Bokeh package
-ROOT_DIR = dirname(dirname(abspath(__file__)))
+ROOT_DIR = Path(__file__).absolute().resolve().parent.parent
 
 __all__ = (
-    'serverdir',
-    'bokehjsdir',
+    "serverdir",
+    "bokehjsdir",
 )
 
 #-----------------------------------------------------------------------------
 # General API
 #-----------------------------------------------------------------------------
 
-def serverdir() -> str:
-    """ Get the location of the server subpackage
+def serverdir() -> Path:
+    """
+    Get the location of the server subpackage.
```

Comment:
Are we changing docstring formats? This is what has been used everywhere else to date
```suggestion
    """ Get the location of the server subpackage.
```